src/cljc/game_of_ur/game/board.cljc

```diff
@@ -124,14 +153,19 @@
       (cond (= 0 roll) (assoc board :turn next-turn)
             (valid-move? board full-move) (assoc next-board :turn next-turn)))))
 
-(defn all-moves [{:keys [turn] :as board} roll]
+(defn all-moves
+  "returns all posible moves with a given roll assuming
+   an empty board"
```

Comment:
Not really an empty board. It just generates moves originating from every cell on the board.

Reply:
"returns all possible moves with a given roll, not taking in count if they are valid or not"  would be right?

Reply:
sounds good to me